README.md

```diff
@@ -6,3 +6,16 @@
 A Golang library for interacting with the Android Debug Bridge (adb).
 
 See [demo.go](cmd/demo/demo.go) for usage.
+
+There is a Makefile at repo root. To run unit tests:
+> make test
+
+For this project, a tool called `stringer` is used to modify several files during `go generate` step in the Makefile:
+```
+devicedescriptortype_string.go
+devicestate_string.go
+internal/errors/errcode_string.go
+```
+Please make sure your environment variable `GOBIN` is correctly set, so that `stringer` can be successfully installed.
```

Comment:
nit: can probably just have a make all target to do everything and just run `make`